docker-compose.yml

```diff
@@ -35,7 +35,9 @@ services:
     volumes:
       - ./binaryData:/webknossos/binaryData
     environment:
-      - POSTGRES_URL=jdbc:postgresql://postgres/webknossos
+      POSTGRES_URL: jdbc:postgresql://postgres/webknossos
+      PGUSER: webknossos
```

Comment:
what do you think about defaulting to `webknossos_user`? I find this easier to understand when seeing it as a string somewhere. Feel free to ignore if you think it would be too unusual.